unity-client/Assets/Scripts/MainScripts/DCL/Components/UUIDComponent/OnPointerEventComponent.cs

```diff
@@ -1,21 +1,36 @@
 using DCL.Controllers;
 using DCL.Models;
 using DCL.Helpers;
+using DCL.Interface;
 using UnityEngine;
 
 namespace DCL.Components
 {
-    public class OnPointerEventComponent : UUIDComponent
+    public class OnPointerEventComponent : UUIDComponent<OnPointerEventComponent.Model>
```

Comment:
The `Component` name in all those family of classes is redundant. I think the code is a bit more readable if we remove it.
```suggestion
    public class OnPointerEvent : UUIDComponent<OnPointerEvent.Model>
```

Reply:
Just checked and it doesn't seem to produce any name collision. So its safe to rename it.

Reply:
Done